Sources/Plasma/CoreLib/HeadSpin.cpp

```diff
@@ -350,7 +354,8 @@ int hsMessageBoxWithOwner(hsWindowHndl owner, const char* message, const char* c
     case IDNO:          return hsMBoxNo;
     default:            return hsMBoxCancel;
     }
-
+#elseif HS_BUILD_FOR_APPLE
+    hsMessageBoxWithOwnerMac(owner, message, caption, kind, icon);
```

Comment:
I think ultimately we're going to want to move the Windows-specific `hsMessageBoxWithOwner` implementation into hsWindows.cpp, and then the macOS one can live in hsMac.mm, and we won't need this indirection here, but this is good for now :+1: